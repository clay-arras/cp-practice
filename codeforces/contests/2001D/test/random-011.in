1
21
15 16 2 6 1 18 10 18 15 21 5 14 24 17 11 8 16 19 10 2 15
