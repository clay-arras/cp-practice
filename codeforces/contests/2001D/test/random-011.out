15
16 2 6 1 18 10 21 5 14 24 17 11 8 19 15
